### PR TITLE
refactor: schoolmint grow notes, comments, strand scoring

### PR DESCRIPTION
**Pull request by cristinabaldor**
# Pull Request

## Summary & Motivation

[//]: # "When merged, this pull request will..."

SchoolMint let us know that score averaged by strand was deprecated so removing,
Bringing in "magic notes" the running record at the top of Walkthrough, O3, and TDT observations as well as measurement comments

## Self-review

- [ ] Update **due date**, **assignee**, and **priority** on our
      [TEAMster Asana Project](https://app.asana.com/0/1205971774138578/1205971926225838)

  _If this is a same-day request, please flag that in Slack_

- [ ] <kbd>Format</kbd> has been run on all modified files

- [ ] Ensure you are using the `union_dataset_join_clause()` macro for queries that employ any
      models from these datasets:
  - deanslist
  - edplan
  - iready
  - pearson
  - powerschool
  - renlearn
  - titan

## Troubleshooting

- [SqlFluff Rules Reference](https://docs.sqlfluff.com/en/stable/rules.html)
- [Trunk](https://teamschools.github.io/teamster/CONTRIBUTING/#trunk)
- [dbt](https://teamschools.github.io/teamster/CONTRIBUTING/#dbt-cloud_1)

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1208138793629837

### CODE REVIEW

File: src/dbt/kipptaf/models/schoolmint/grow/staging/stg_schoolmint_grow__observations__observation_notes.sql
Comment (cbini) [inline src/dbt/kipptaf/models/schoolmint/grow/staging/stg_schoolmint_grow__observations__observation_notes.sql:1]: rename to `stg_schoolmint_grow__observations__magic_notes`

File: src/dbt/kipptaf/models/schoolmint/grow/staging/stg_schoolmint_grow__observations__observation_notes.sql [hunk 1]
```diff
@@ -0,0 +1,11 @@
+select
+    o.observation_id,
+
+    mn.`column` as notes_column,
+    mn.shared as notes_shared,
+    mn.text as notes_text,
+    mn.created as notes_created,
```
Comment (cbini) [inline src/dbt/kipptaf/models/schoolmint/grow/staging/stg_schoolmint_grow__observations__observation_notes.sql:7]: remove `notes_` from these aliases

File: src/dbt/kipptaf/models/schoolmint/grow/intermediate/int_schoolmint_grow__observations.sql [hunk 2]
```diff
@@ -12,6 +11,8 @@ select
     o.academic_year,
     o.is_published,
 
+    oon.notes_text as observation_notes,
```
Comment (cbini) [inline src/dbt/kipptaf/models/schoolmint/grow/intermediate/int_schoolmint_grow__observations.sql:14]: rename `magic_notes_text`

**Review by cbini — CHANGES_REQUESTED** [review]

File: src/dbt/kipptaf/models/schoolmint/grow/intermediate/int_schoolmint_grow__observations.sql (hunk 2 above)
Comment (cristinabaldor) [reply src/dbt/kipptaf/models/schoolmint/grow/intermediate/int_schoolmint_grow__observations.sql:14]: the alias or the original column name?

**Review by cbini — APPROVED** [review]

File: src/dbt/kipptaf/models/schoolmint/grow/intermediate/int_schoolmint_grow__observations.sql (hunk 2 above)
Comment (cbini) [reply src/dbt/kipptaf/models/schoolmint/grow/intermediate/int_schoolmint_grow__observations.sql]: the original column name should be `text`, the alias should be `magic_notes_text`. downstream from here, feel free to rename to whatever, but on staging/int I like to preserve the original names as much as possible

**Review by cbini — APPROVED** [review]

**Pull request merged**